typescript/cli/src/context/context.ts

```diff
@@ -145,7 +159,12 @@ function getRegistry(
     .map((uri, index) => {
       const childLogger = logger.child({ uri, index });
       if (isHttpsUrl(uri)) {
-        return new GithubRegistry({ uri, logger: childLogger });
+        const options: GithubRegistryOptions = {
+          uri,
+          logger: childLogger,
+        };
+        options.proxy ??= proxyUri;
+        return new GithubRegistry(options);
```

Comment:
Why split out the options object like this? Could you instead have added a `proxy: proxyUri` field to the way it was before?

Reply:
fixed